solutions/standard/outputs.tf

```diff
@@ -0,0 +1,28 @@
+##############################################################################
+# Outputs
+##############################################################################
```

Comment:
More of a comment than review statement: let's make sure that in the outputs for this DA (which will show up in the Projects outputs) contain everything that would be needed to complete the setup of the VPN client software. I can't tell if anything is missing here, just a reminder to make sure we have everything.